configs/ocrnet-hrnet+psa/README.md

```diff
@@ -0,0 +1,17 @@
+# PSA: Polarized Self-Attention: Towards High-quality Pixel-wise Regression
+
+## Reference
+
+> Huajun Liu, Fuqiang Liu, Xinyi Fan and Dong Huang. "Polarized Self-Attention: Towards High-quality Pixel-wise Regression." arXiv preprint arXiv:2107.00782v2 (2021).
+
+## Performance
+
+### Cityscapes
+
+| Model            | Backbone    | Resolution | Training Iters | mIoU   | mIoU (flip) | mIoU (ms+flip) | Links           |
+| ---------------- | ----------- | ---------- | -------------- | ------ | ----------- | -------------- | --------------- |
+| OCRNet-HRNet+psa | HRNETV2_W48 | 1024x2048  | 80000          | 84.55% | 84.73%      | 84.92%         | model\|log\|vdl |
+
+### Notes
+
+Because the authors use **train + val set for training** and val set for evaluating, we only use the train set for training, the final accuracy is lower than that in paper.
```

Comment:
实际上从两篇原文中很难直接敲定作者用了train+val训练并且用val测试（虽然从文中有限的表述上来看，作者这样做的可能性很大），严谨起见，我们还是不要这样指责。我认为有一点是可以肯定的，就是作者提供的代码跑不通训练（代码和说明都是不完整的），无法复现。所以我建议采用如下的表述：

Since we cannot reproduce the training results from [the authors' official repo](https://github.com/DeLightCMU/PSA), we follow the settings in the original paper to train and evaluate our models, and the final accuracy is lower than that reported in the paper.

你同意我的观点吗？

Reply:
同意，并修改

Reply:
同意，已修改